src/WebJobs.Extensions.DurableTask/EtwEventSource.cs

```diff
@@ -400,6 +400,19 @@ private EtwEventSource()
         {
             this.WriteEvent(224, TaskHub, AppName, SlotName, FunctionName, InstanceId, Reason, FunctionType, ExtensionVersion, IsReplay);
         }
+
+        [Event(225, Level = EventLevel.Informational)]
```

Comment:
Yes, that's right: I want to capture this in linux and, by default, we don't capture verbose logs in there.

Another option is: we could make an exception for this event in our "Verbose-filtering" logic for the linux-logger.

However, especially that this event occurs only once per replay, I'd argue it is more fitting for "informational" than "verbose"

Reply:
More than just how frequently telemetry is written, the spirit of "information" is that it contains traces that tell the customer _what is happening_. This goes beyond that by dumping a bunch of internal protocol data that is intended only for us. This is why I feel like "Informational" is not really appropriate. I think it would be better to have the Linux logger handle this case specifically. However, that doesn't necessarily have to be part of this PR.

Reply:
I see what you mean, I've changed it to `Verbose` and I'll handle this special case with the linux logger in another PR